src/util/tree.js

```diff
@@ -7,6 +7,9 @@ export let Tree = F.tree(
 )
 
 export let flatten = Tree.flatten(F.propTreePath('key'))
+// contexture trees use a slash encoder to disambiguate it from lodash object paths
+// if you saw 'root.criteria.filters' you might think that’s an object structure, but the actual object structure in that example is 'children[0].children[0]'
+// so 'root/criteria/filters' is the encoding
```

Comment:
this comment formatting is terrible but I'll let it slide

Reply:
Blindly copied and pasted from our slack conversation 😅